ert3/console/_console.py

```diff
@@ -1,17 +1,30 @@
-import ert3
-
 import argparse
-from pathlib import Path
+import pathlib
+import shutil
 import sys
+from pathlib import Path
+
+import pkg_resources as pkg
 import yaml
 
+import ert3
+
 _ERT3_DESCRIPTION = (
     "ert3 is an ensemble-based tool for uncertainty studies.\n"
     "\nWARNING: the tool is currently in an extremely early stage and we refer "
     "all users to ert for real work!"
 )
 
 
+def _build_init_argparser(subparsers):
+    init_parser = subparsers.add_parser("init", help="Initialize an ERT3 workspace")
+    init_parser.add_argument(
+        "--example",
+        help="Name of the example that would be copied "
+        "to the working directory and initialised",
```

Comment:
It would be really neat to list the available examples here. If you choose to not address it in this PR, it would be nice if you could create an issue and write in the PR description that you defer that to later efforts :)

Reply:
Make sense 👍

Reply:
I decided to add list of available examples in this PR